about/release_policy.rst

```diff
@@ -82,8 +82,11 @@ on GitHub.
 +--------------+----------------------+--------------------------------------------------------------------------+
 | **Version**  | **Release date**     | **Support level**                                                        |
 +--------------+----------------------+--------------------------------------------------------------------------+
-| Godot 4.3    | June 2024            | |unstable| *Development.* Receives new features, usability and           |
-| (`master`)   | (estimate)           | performance improvements, as well as bug fixes, while under development. |
+| Godot 4.4    | To be announced      | |unstable| Receives new features, usability and                          |
+|              |                      | performance improvements, as well as bug fixes, while under development. |
```

Comment:
```suggestion
| Godot 4.4    | To be announced      | |unstable| *Development.* Receives new features, usability and           |
| (`master`)   | (estimate)           | performance improvements, as well as bug fixes, while under development. |
```
Just change the version and date, otherwise keep the same as before